R/nmbayes-tail.R

```diff
@@ -0,0 +1,68 @@
+#' Inspect first and last lines of select files from a NONMEM Bayes model
+#'
+#' `tail_lst()` and `tail_output()` provide [bbr::tail_lst()] and
+#' [bbr::tail_output()] methods for `bbi_nmbayes_model` objects. Compared to the
+#' methods for `bbi_nonmem_model` objects, the key difference is that these
+#' operate on the `*.lst` or `OUTPUT` file of each chain submodel.
+#'
+#' @inheritParams bbr::check_file
+#' @param .return If `TRUE`, return the head and tail results for each chain.
+#'   See \sQuote{Value} section for more details.
+#' @param ... Arguments passed to method call on each chain submodel.
+#' @return If `.return` is `TRUE`, return a list of character vectors, with an
+#'   element for each chain. The character vector is comprised of the first and
+#'   last lines of the file under a chain subdirectory. For `tail_output()`, the
+#'   element will be `NULL` if 1) the model has started to run but hasn't yet
+#'   created the `OUTPUT` file or 2) the model has finished running.
+#' @seealso [chain_paths()] for constructing chain submodels paths
+#' @name nmbayes_tail
+#' @aliases nmbayes_tail_lst nmbayes_tail_output
+NULL
+
+#' @rdname nmbayes_tail
```

Comment:
I like this pattern, so that the user can call `?nmbayes_tail` to get these docs. It always bothers me (for methods like this) that a `bbr.bayes` user calling `?tail_lst` will get the `bbr` docs and would need to know the class name and call `?tail_lst.bbi_nmbayes_model` to see these docs. 

The discoverability of the `?nmbayes_tail` pattern isn't obvious to me, but once you know it, it's easy to use.

Reply:
I had forgotten that we already had this for `submit_model` ([Stan](https://github.com/metrumresearchgroup/bbr.bayes/blob/7825c6f9430a4048b149f9a71b011f4f3c48696a/man/stan_submit_model.Rd) and [nmbayes](https://github.com/metrumresearchgroup/bbr.bayes/blob/7825c6f9430a4048b149f9a71b011f4f3c48696a/man/nmbayes_submit_model.Rd)) and other methods too. Well done. We should probably mention this in the Expo and other relevant places (vignettes?).

Reply:
> We should probably mention this in the Expo and other relevant places (vignettes?).

For the vignettes, I added a mention at gh-111.

I also created an issue for checking whether we should use the prefix more consistently in the docs: gh-131